searchchunkswidget.cpp

```diff
@@ -0,0 +1,185 @@
+#include "searchchunkswidget.h"
+#include "ui_searchchunkswidget.h"
+
+#include "./chunkcache.h"
+#include "./chunkmath.h"
+#include "./range.h"
+#include "./rectangleinnertoouteriterator.h"
+
+#include <QVariant>
+#include <QTreeWidgetItem>
+#include <boost/noncopyable.hpp>
```

Comment:
Is it necessary to use BOOST here?
I do not have boost installed and therefore it does not compile anymore.
Uncommenting this line let it compile again.

Reply:
Seems unnecessary.  I removed the include.. and definitely agree that we should never use boost if we can help it.

Reply:
Thanks for spotting and removing this. Seems to be a leftover that I didn't notice as I have boost installed. The boost::noncopyable is a very small class which at itself doesn't justify to add the dependency to boost - I agree.